openwisp_monitoring/check/base/models.py

```diff
@@ -15,7 +15,14 @@
 
 class AbstractCheck(TimeStampedEditableModel):
     name = models.CharField(max_length=64, db_index=True)
-    active = models.BooleanField(default=True, db_index=True)
+    is_active = models.BooleanField(
+        _('active'),
+        default=True,
+        db_index=True,
+        help_text=_(
+            'whether the check should be run, related metrics be collected and alerts sent'
```

Comment:
`related metrics be collected` doesn't sound correct to me, I think it should be `related metrics collected`